src/GraphQL.MicrosoftDI/DynamicScopedFieldResolver.cs

```diff
@@ -26,5 +27,6 @@ public DynamicScopedFieldResolver(IFieldResolver resolver)
     }
 
     /// <inheritdoc/>
+    [AllowedOn<IObjectGraphType>]
```

Comment:
This is just a `IFieldResolver` implementation, so I don't think we need to set it here, right?

Reply:
Yeah, you are right